openstack/clustering/v1/policytypes/testing/fixtures.go

```diff
@@ -69,6 +71,67 @@ const PolicyTypeBody = `
 }
 `
 
+const PolicyTypeDetailBody = `
+{
+    "policy_type": {
+        "name": "senlin.policy.affinity-1.0",
+        "schema": {
+            "availability_zone": {
+                "description": "Name of the availability zone to place the nodes.",
+                "required": false,
+                "type": "String",
+                "updatable": false
+            },
+            "enable_drs_extension": {
+                "default": false,
+                "description": "Enable vSphere DRS extension.",
+                "required": false,
+                "type": "Boolean",
+                "updatable": false
+            },
+            "servergroup": {
+                "description": "Properties of the VM server group",
+                "required": false,
+                "schema": {
+                    "name": {
+                        "description": "The name of the server group",
+                        "required": false,
+                        "type": "String",
+                        "updatable": false
+                    },
+                    "policies": {
+                        "constraints": [
+                            {
+                                "constraint": [
+                                    "affinity",
+                                    "anti-affinity"
+                                ],
+                                "type": "AllowedValues"
+                            }
+                        ],
+                        "default": "anti-affinity",
+                        "description": "The server group policies.",
+                        "required": false,
+                        "type": "String",
+                        "updatable": false
+                    }
+                },
+                "type": "Map",
+                "updatable": false
+            }
+        },
+        "support_status": {
+            "1.0": [
+                {
+                    "status": "SUPPORTED",
+                    "since": "2016.10"
+                }
+            ]
+        }
+    }
+}
```

Comment:
Side note: normally we don't like to use test fixtures pulled from the API docs. This is for two reasons:

1. It might be an old/stale response (we've seen this with some of the networking API fixtures)
2. It might not exercise all fields we want to test

Admittedly this is something we (I) often forget to verify, though. Since you folks are more familiar with Senlin/Clustering than me, I'll defer to your judgement on whether the API doc fixtures are usable or if you should use fixtures generated from your environment (usually done by using the `--debug` flag with the command-line tools).

Reply:
I think the API doc is up-to-date, but I will double-check that.